config/cypress.yml

```diff
@@ -3,7 +3,7 @@ effective_date:
   month: 12
   day: 31
 
-version: 3.0.0
+version: 3.0.1
```

Comment:
We should also bump the version number in the build scripts, so we can build new VMs, etc.